packages/sw-runtime-caching/test/sw/cache-first.js

```diff
@@ -0,0 +1,33 @@
+importScripts('setup.js');
+
+describe('Test of the CacheFirst handler', function() {
+  const CACHE_NAME = location.href;
+  const REQUEST_WRAPPER = new goog.runtimeCaching.RequestWrapper({cacheName: CACHE_NAME});
+  const CACHE_FIRST = new goog.runtimeCaching.CacheFirst({requestWrapper: REQUEST_WRAPPER, waitOnCache: true});
+  const COUNTER_URL = new URL('/__echo/counter', location).href;
+
+  let cache;
+  let initialCachedResponse;
+
+  before(() => {
+    const event = new FetchEvent('fetch', {request: new Request(COUNTER_URL)});
+    return caches.delete(CACHE_NAME)
```

Comment:
Looking through this, I'd just consolidate everything into a single unit test and use before() to delete any existing caches.